packages/atomic/cypress/integration/result-list/result-list.cypress.ts

```diff
@@ -5,154 +5,198 @@ import {
   ResultListSelectors,
 } from './result-list-selectors';
 import {generateComponentHTML, TestFixture} from '../../fixtures/test-fixture';
-import {addResultList, buildTemplateWithSections} from './result-list-actions';
+import {
+  addFoldedResultList,
+  addResultList,
+  buildTemplateWithSections,
+} from './result-list-actions';
 import * as CommonAssertions from '../common-assertions';
-
-describe('Result List Component', () => {
-  it('should load', () => {
-    new TestFixture().with(addResultList()).init();
-    ResultListSelectors.result().should('have.length.above', 0);
-  });
-
-  describe('when no first search has yet been executed', () => {
-    beforeEach(() => {
-      new TestFixture()
-        .withoutFirstAutomaticSearch()
-        .with(addResultList())
-        .init();
-    });
-
-    it('should render placeholder components', () => {
-      ResultListSelectors.placeholder().should('be.visible');
-    });
-  });
-
-  describe('when an initial search is executed', () => {
-    it('should render the correct number of results', () => {
-      new TestFixture().with(addResultList()).init();
-      ResultListSelectors.result().should('have.length', 10);
-    });
-  });
-
-  describe('when multiple searches are executed', () => {
-    beforeEach(() => {
-      new TestFixture()
-        .with(addResultList())
-        .withElement(generateComponentHTML(pagerComponent))
-        .init();
+import {
+  foldedResultListComponent,
+  FoldedResultListSelectors,
+} from './folded-result-list-selectors';
+
+const foldedResultListConfig = {
+  componentSelectors: FoldedResultListSelectors,
+  componentTag: foldedResultListComponent,
+  addResultFn: addFoldedResultList,
+  title: 'Folded result List Component',
+};
+
+const resultListConfig = {
+  componentSelectors: ResultListSelectors,
+  componentTag: resultListComponent,
+  addResultFn: addResultList,
+  title: 'Result List Component',
+};
+
+const configs: Array<typeof foldedResultListConfig> = [
```

Comment:
instead of adding the type `Array<typeof foldedResultListConfig>`, you may be able to use `as const`:
```ts
const configs = [
  ...
] as const;
```

Reply:
silly typescript https://stackoverflow.com/a/66993654/8454914

Reply:
and great suggestion